tt_metal/hw/ckernels/wormhole_b0/metal/common/chlkc_list.h

```diff
@@ -14,6 +14,7 @@ using namespace ckernel;
 
```

Comment:
@yugaoTT can you add all these changes to blackhole as well?

Reply:
I'd like to add it in another PR, after some op issues are solved for BH

Reply:
ok as long as you make an issue for BH just to track missing features

Reply:
Thanks. created issue here:
https://github.com/tenstorrent/tt-metal/issues/13004